lib/phoenix_html/form.ex

```diff
@@ -400,6 +400,11 @@ defmodule Phoenix.HTML.Form do
 
   We will explore all them below.
 
+  Note that if you are using Phoenix LiveView, `form_for/4` is no longer the
```

Comment:
This could be confusing. One could be using phoenix 1.7 (which depends on LiveView for html), but not run any "LiveViews". They still want to use `<.form>`. 

Maybe `if you are using HEEx/function components …` would be closer? Both heex and function component mean the `phoenix_live_view` package is available, without the need of using "LiveViews".

Reply:
I agree that it's confusing, but it's worth noting that the language "if you are using Phoenix LiveView" already appears in this file on L348:

```elixir
  def form_for(form_data, action, options) when is_list(options) do
    IO.warn(
      "form_for/3 without an anonymous function is deprecated. " <>
        "If you are using Phoenix.LiveView, use the new Phoenix.Component.form/1 component"
    )

    %{Phoenix.HTML.FormData.to_form(form_data, options) | action: action}
  end
```

So if it's going to be changed, it should be changed in both places.